test/classes/eik-config/mappings.test.js

```diff
@@ -14,7 +14,7 @@ const validEikConfig = {
 
 const baseDir = join(__dirname, '../../fixtures');
 
-test('mappings - directory given', async (t) => {
+test('mappings - directory given', { only: true }, async (t) => {
```

Comment:
Stragler `only` from debugging
```suggestion
test('mappings - directory given', async (t) => {
```